src/plugins/analysis/strings/internal/string_eval.py

```diff
@@ -0,0 +1,132 @@
+"""
+evaluate relevance of strings
+
+Credits:
+Original version by Paul Schiffer created during Firmware Bootcamp WT16/17 at University of Bonn
+Refactored and improved by Fraunhofer FKIE
+"""
```

Comment:
I don't think that doc comment is that useful.

Reply:
Removed the doc part and added a more informative docstring to the function instead.